docs/rules/no-namespace.md

```diff
@@ -0,0 +1,95 @@
+# Disallows the use of `internal modules` and `namespaces`.
+
+Internal modules (`module foo {}`) and namespaces (`namespace foo {}`) are considered outdated 
+ways to organize TypeScript code. ES2015 module syntax is now preferred.   
+
+This rule still allows the use of external modules (`declare module 'foo' {}`).
+
+## Rule Details
+
+This rule aims to standardise the way TypeScript modules are declared.
+
+## Options
```

Comment:
It does not feel right to have a rule called "no-" and then have the exceptions enabled by default.

Please change the defaults for both exceptions to `false`.